docs/feature_flags.md

```diff
@@ -40,7 +40,17 @@ with more recent data.
 
 More details can be found [here](querying/basics.md#offset-modifier).
 
-## Remote Write Receiver
+# Evaluation time aligned subqueries in PromQL
+
+`--enable-feature=promql-eval-aligned-subqueries`
+
+As opposed to the normal `[x:y]` subquery syntax, which aligns with multiples of the subquery's
+interval as a [performance optimization](https://www.robustperception.io/promql-subqueries-and-alignment),
+the expiremental syntax `[x::y]` aligns with the evaluation time no matter what it is. This gives you
```

Comment:
expiremental -> experimental

Reply:
I see this one resolved, but the typo is still there?